docs/source/usersguide/variance_reduction.rst

```diff
@@ -0,0 +1,144 @@
+   via OpenMC in the :ref:`multigroup materials <create_mgxs>` user guide, and a
+   specific example of generating cross section data for use with random ray in
+   the :ref:`random ray MGXS guide <mgxs_gen>`. 
+
+2. Make a copy of your continuous energy python input file. You'll edit the new
+   file to work in multigroup mode with random ray for producing weight windows.
+
+3. Adjust the material definitions in your new multigroup python file to utilise
+   the multigroup cross sections instead of nuclide-wise continuous energy data.
+   There is a specific example of making this conversion in the random ray in
+   the :ref:`random ray MGXS guide <mgxs_gen>`.
+
+4. Configure OpenMC to run in random ray mode (by adding several standard random
+   ray input flags and settings to the :attr:`openmc.Settings.random_ray`
+   dictionary). More information can be found in the  :ref:`Random Ray User
+   Guide <random_ray>`. 
+
+5. Add in an :class:`openmc.WeightWindowGenerator` in a similar manner as for
+   MAGIC generation with Monte Carlo, though with the :attr:`method` attribute
+   set to ``fw_cadis``::
+
+    # Define weight window spatial mesh
+    ww_mesh = openmc.RegularMesh()
+    ww_mesh.dimension = (10, 10, 10)
+    ww_mesh.lower_left = (0.0, 0.0, 0.0)
+    ww_mesh.upper_right = (100.0, 100.0, 100.0)
+
+    # Create weight window object and adjust parameters
+    wwg = openmc.WeightWindowGenerator(method='fw_cadis', mesh=ww_mesh, max_realizations=settings.batches)
+
+    # Add generator to openmc.settings object
+    settings.weight_window_generators = wwg    
+
+
+.. warning::
+    If using FW-CADIS weight window generation, ensure that the selected weight window mesh does not subdivide any cells in the problem. In the future, this restriction is intended to be relaxed, but for now subdivision of cells by a mesh tally will result in undefined behavior.
+
+6. When running your multigroup random ray input deck, OpenMC will automatically
+   run a forward solve followed by an adjoint solve, with a
+   ``weight_windows.h5`` file generated at the end. The ``weight_windows.h5``
+   file will contain FW-CADIS generated weight windows. This file can be used in
+   identical manner as one generated with MAGIC, as described below.
+
+--------------------
+Using Weight Windows
+--------------------
+
+To use a "weight_windows.h5" weight window file with OpenMC's Monte Carlo
+solver, the python input just needs to load the h5 file::
+
+    settings.weight_window_checkpoints = {'collision': True, 'surface': True}
+    settings.survival_biasing = False
+    settings.weight_windows = openmc.hdf5_to_wws()
+    settings.weight_windows_on = True
+
+Make sure that the :class:`openmc.WeightWindowGenerator` is not present in the
+file when loading existing weight windows, so as to avoid added costs of
+generating weight windows again and overwriting the original weight window file.
```

Comment:
I might just be tired but I'm having trouble parsing this sentence :sweat_smile:

Reply:
I've updated this -- let me know if it makes more sense now.